examples/asr/asr_webapp/README.rst

```diff
@@ -0,0 +1,70 @@
+**ASR SERVICE **
+================
+
+Usage
+-----
+
+A simple service that will load a pre-trained model from NeMo's ASR collection and after uploading some files, will transcribe them. Any pre-trained model can be selected and if the local / docker environment supports GPUs then it will be possible to transcribe the audio segments with GPU.
+
+All uploaded files are immediately deleted after transcriptions are obtained.
+
+ .. note::
+
+    When using gunicorn, you might notice that each pretrained checkpoint takes a long time for the first transcription. This is because that worker is instantiating the model into memory and then moving the model onto the GPU. For large models, this step might take a significant amount of time. However, this step is cached, and subsequent transcription requests should be much faster (especially if a GPU is utilized).
+
+There are three options to run the ASR Service to transcribe audio -
+
+Local Setup
+-----------
+
+1) Install all the dependencies (``pip install -r requirements.txt``). Note : It will utilize the latest branch set in the requirements and will override current nemo installation.
```

Comment:
Where are these files located? Maybe give the path to the folder?

Reply:
They are local to the service directory. Good point, ill note it somewhere.